lib/options.js

```diff
@@ -64,6 +64,11 @@ var defaults = {
     // array expressions, functions calls and function definitions pass true
     // for this option.
     trailingComma: false,
+
+    // There are 2 supported syntaxes (`,` and `;`) in Flow Object Types;
+    // The use of commas is in line with the more popular style and matches
+    // how objects are defined in JS, making it a bit more natural to write.
+    flowUsesCommas: true,
```

Comment:
Oh actually, would it be better to change the default to `false` to avoid having a breaking change? @benjamn